Hippo.Web/Services/BulkLoadService.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Hippo.Core.Data;
+using Hippo.Core.Domain;
+using Microsoft.EntityFrameworkCore;
+using Serilog;
+using static Hippo.Core.Domain.Account;
+
+namespace Hippo.Web.Services
+{
+    public interface IBulkLoadService
+    {
+        Task<int> Load(Cluster cluster, string data);
+    }
+
+    public class BulkLoadService : IBulkLoadService
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly IIdentityService _identityService;
+
+        public BulkLoadService(AppDbContext dbContext, IIdentityService identityService)
+        {
+            _dbContext = dbContext;
+            _identityService = identityService;
+        }
+
+
+
+        public async Task<int> Load(Cluster cluster, string data)
+        {
+            var count = 0;
+            var pairs = data.Split(",");
+
+            Log.Information($"Starting bulk load for {pairs.Count()} accounts");
+
+            //Create missing users
+            var kerbs = new List<string>();
+            var kerbsInDb = new List<string>();

```

Comment:
This should probably be a `HashSet<string>` if you're going to be calling `.Contains(...)` on it in a loop.